pkgs/by-name/rt/rtg-tools/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  config,
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  jdk,
+  ant,
+  git,
+  unzip,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rtg-tools";
+  version = "3.12.1";
+
+  src = fetchFromGitHub {
+    owner = "RealTimeGenomics";
+    repo = "rtg-tools";
+    rev = version;
+    sha256 = "sha256-fMrrjrgaGxBVxn6qMq2g0oFv6qtfhZcQlkvv1E9Os6Y=";
+  };
+
+  nativeBuildInputs = [
+    ant
+    jdk
+    git
+    unzip
+  ];
+
+  buildPhase = ''
+    ant rtg-tools.jar
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp build/rtg-tools.jar $out/bin/RTG.jar
+    cp installer/rtg $out/bin/
+    cp installer/rtg $out/bin/
+  '';
+
+  postFixup = ''
+    # Use a location outside nix (must be writable)
+    sed -i '/USER_JAVA_OPTS=$RTG_JAVA_OPTS/a  mkdir -p $HOME/.config/rtg-tools'  $out/bin/rtg
+    sed -i 's|$THIS_DIR/rtg.cfg|$HOME/.config/rtg-tools/rtg.cfg|g' $out/bin/rtg
+    # Use nix java
+    sed -i 's|RTG_JAVA="java".*|RTG_JAVA="${jdk}/lib/openjdk/bin/java"|' $out/bin/rtg
```

Comment:
Since we just do simple string replacement here, we should use substituteInPlace with --replace-fail to notice, if the pattern no longer matches.

Reply:
I've switched to substituteInPlace, except for appending to a string that I only know how to do it with sed :/